legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/test/runner/service/ServiceTestRunner.java

```diff
@@ -67,6 +67,7 @@
 import org.finos.legend.engine.protocol.pure.v1.model.valueSpecification.raw.PureList;
 import org.finos.legend.engine.shared.core.ObjectMapperFactory;
 import org.finos.legend.engine.shared.core.operational.Assert;
+import org.finos.legend.engine.shared.core.operational.logs.LoggingEventType;
```

Comment:
Do you intend to update MappingTestRunner ?

Reply:
It seems like MappingTestRunner is only used for tests and not when the server is live. I think errors coming from there would not be added to the metrics properly. I think we should leave MappingTestRunner as it is.